packages/convertim/.gitignore

```diff
@@ -0,0 +1,4 @@
+/.php_cs.cache
```

Comment:
We have also `/.idea` in most of our packages

Reply:
added – 66b953cb0f